schemas/data/adhoc.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/data/adhoc",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adhoc Schema",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "description":
+    "Used to indicate the behaviour of Adhoc data semantic when composed into data Schemas. The Adhoc data semantic scope is limited to just the local schema, and no compatibility can be assumed between schemas.",
```

Comment:
just wondering if it is a good idea to be specific and mention that field values can not be assumed to be standardized/normalized with existing schemas.

Reply:
@rkpandey  The description currently says ""The Adhoc data semantic scope is limited to just the local schema, and no compatibility can be assumed between schemas."" Is that ok?